docs/RUBRIC.md

```diff
@@ -7,33 +7,40 @@ This Rubric is based on the [user stories](https://github.com/opencontainers/wg-
 The Rubric will be updated based on consensus reached in the community meetings.
 
 Key:
+
 - F: Filtering
 - B: Backwards Compatibility
 - C: Content Management
 
 |User Stories|Proposal A|Proposal B|Proposal C|Proposal D|
 |--- |--- |--- |--- |--- |
-|**F.1**| | | | |
-|**F.2**| | | | |
-|**F.3**| | | | |
-|**F.5**| | | | |
-|**F.6**| | | | |
-|**B.1**| | | | |
-|**B.2**| | | | |
-|**B.3**| | | | |
-|**B.4**| | | | |
-|**B.5**| | | | |
-|**B.6**| | | | |
-|**C.1**| | | | |
-|**C.2**| | | | |
-|**C.3**| | | | |
-|**C.4**| | | | |
-|**C.5**| | | | |
-|**C.6**| | | | |
-|**C.7**| | | | |
-|**C.8**| | | | |
-|**C.9**| | | | |
-|**C.10**| | | | |
-|**C.11**| | | | |
-|**C.12**| | | | |
-|**C.13**| | | | |
+|**F.1**| | | | Yes |
+|**F.2**| | | | Yes |
+|**F.3**| | | | Yes |
```

Comment:
Since accomplishing F.2 and F.3 requires looking at the annotations as well, wouldn't this also be a Partial (1)?

Reply:
The type is defined in the tag or in the index that was already pulled for the image. So the client should have all the information they need without first fetching each artifact manifest.